### ISSUE TITLE
S607 doesn't validate for tuple values

### ISSUE THREAD

**Issue by nelsyeung** [astral-sh/ruff#24075]
### Summary

When using a tuple (i.e., `(...,)`), it doesn't give S607 ([Playground](https://play.ruff.rs/27cff869-b233-4cdf-b861-f77272ae22ea)):
```python
import subprocess

subprocess.run(("echo", "foo"))
```

but using a list (i.e., `[..., ...]`) does ([Playground](https://play.ruff.rs/1eaf82e1-ba38-4e31-a608-b025b5210004)):
```python
import subprocess

subprocess.run(["echo", "foo"])
```

~~I mentioned multiple values, because if there's only one element in the tuple, it'll give S607 ([Playground](https://play.ruff.rs/6404aa88-6d82-4672-aec0-0577e0623a4b))~~:

```python
import subprocess

subprocess.run(("echo"))
```

### Version

0.15.7

**Comment by dylwil3** [comment]
I think all tuples fail here - your last example isn't a tuple it's just an expression with parentheses around it. The rule again doesn't trigger if you add a comma:

```python
import subprocess

subprocess.run(("echo",))
```

But good catch, this looks like a bug!
